src/app/clusters/scenes-server/BUILD.gn

```
@@ -13,30 +13,33 @@
 # limitations under the License.
 import("//build_overrides/chip.gni")
 
-static_library("scenes") {
-  output_name = "libCHIPScenes"
-
-  sources = [
+source_set("scene-table") {
```

```suggestion
source_set("scenes-table") {
```

Just for consistency, as the cluster directory is `scenes-server` . So either `scenes` or `scene` everywhere

---

I don't think we can avoid inconsistency here; the class itself (the primary entry point for this package) is called `SceneTable` (& was so before these changes) - it makes sense to have the package match the class name, `scene-table` - and as we now have a target without the plural, it makes sense to prefer this

One can probably make the argument that a cluster for managing scenes & thus the plural, whereas its modules (the table for scenes, common libraries for scenes, etc.) should be singular